plugin/github.com.hashicorp.go.kms.wrapping.plugin.v2.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+
+option go_package = "github.com/hashicorp/go-kms-wrapping/v2/plugin;plugin";
+
+package github.com.hashicorp.go.kms.wrapping.plugin.v2;
+
+import "github.com.hashicorp.go.kms.wrapping.v2.types.proto";
+
+message TypeRequest {}
+
+message TypeResponse {
+	string type = 10;
```

Comment:
How come we're starting the index at 10 (and sometimes 20 for other messages)?

Reply:
I tend to leave space so that we can insert more parameters in between later. I'm actually not sure why some start with 20, I've updated all to 10.

If you feel strongly I can start them all at 1.

Reply:
Ah I remember -- I just kept `options` as field 20 across all messages for consistency. Just a style, but can use something else if you prefer.